app/views/admin/shared/_nested_sidebar.html.erb

```diff
@@ -0,0 +1,29 @@
+<% menu_items.each do |group_name, group| %>
+  <% if group[:children].length == 1 %>
+    <a class="crayons-link crayons-link--block <%= "crayons-link--current" if (deduced_controller(request) == group[:children][0][:controller]) %>"
+       href="/admin/<%= group[:children][0][:controller] %>"
+    >
+      <%= inline_svg_tag("#{group[:svg]}", aria: true, class: "dropdown-icon crayons-icon", title: "#{group_name}") %>
+      <%= display_name(group_name) %>
+    </a>
+  <% else %>
+    <div class="crayons-link crayons-link--block cursor-pointer <%= "crayons-link--current" if (deduced_scope(request) == group_name.to_s) %>"
+      data-toggle="collapse"
+      data-target="#<%= group_name %>"
+      aria-expanded="<%= (deduced_scope(request) == group_name.to_s).to_s %>"
+      aria-controls="<%= group_name %>">
+      <%= inline_svg_tag("#{group[:svg]}", aria: true, class: "dropdown-icon crayons-icon", title: "#{group_name}") %>
+      <%= display_name(group_name) %>
+    </div>
```

Comment:
(refactor / blocking) - As this item is a `div` rather than an interactive element, it's not currently possible by keyboard to expand/collapse the menus - can we change to a button? If it helps, there's a nice example of this in the [WAI ARIA practices example](https://www.w3.org/TR/wai-aria-practices/examples/disclosure/disclosure-navigation.html)

It's also best practice to immediately focus the first link in the list after the expanded items are expanded - is it possible to trigger this in the controller when we toggle the list?
 
Nice one with the `aria-expanded` and `controls` 👍

Reply:
> can we change to a button? 
✅

>is it possible to trigger this in the controller when we toggle the list?
@lisasy brought this up [here for different reasons](https://github.com/forem/forem/pull/12936#pullrequestreview-606770573). I'm working on a bigger change for the interaction design, and I'll incorporate this change in this follow up PR.